### ISSUE TITLE
Provide streams examples in Learn->manipulating-files->"Reading files with Node.js" article

### ISSUE THREAD

**Issue by henrik-jensen** [nodejs/nodejs.org#6921]
## 1. What I expected to happen.

The [Learn](https://nodejs.org/en/learn) page [Reading files with Node.js](https://nodejs.org/en/learn/manipulating-files/reading-files-with-nodejs) ends with:
> ...
> All three of fs.readFile(), fs.readFileSync() and fsPromises.readFile() read 
> the full content of the file in memory before returning the data.
> This means that big files are going to have a major impact on your memory 
> consumption and speed of execution of the program.
> In this case, a better option is to read the file content using streams.
> 

I would have expected a link to a streams example solving the same problem or at least a link to the streams api (and possibly, hopefully not true, a clarification that this is neck-beard territory
[![Ken_Thompson_and_Dennis_Ritchie--1973](https://github.com/user-attachments/assets/a7960218-8090-478d-9c7a-3b763c91c2c1 "Ken Thompson (left) and Dennis Ritchie (right), inventors of, and Unix/C gurus")](https://en.wikipedia.org/wiki/Unix)
and too advanced for you, my young padawan 😀)

## 2. Your reason (if possible, images and videos are welcome).
Reading data from unknown and potentially unreliable sources is a common task in computing, so why not give those young padawans a good head start by teaching the recommended techniques?

## 3. What I plan to do (Optional but better).
Study the [Streams API](https://nodejs.org/api/stream.html) and see if I can come up with a pull request for the docs, but don't get your hopes too high,- I consider myself still a padawan regarding node.js.

## 3b. What I believe should be updated (at the request of @ovflowd).
Provide a small example in [Reading files with Node.js](https://nodejs.org/en/learn/manipulating-files/reading-files-with-nodejs), if possible, that uses the [Streams API](https://nodejs.org/api/stream.html) as hinted in the last sentence of [Reading files with Node.js](https://nodejs.org/en/learn/manipulating-files/reading-files-with-nodejs): _"...In this case, a better option is to read the file content using streams."_

**Comment by ovflowd** [comment]
I am not sure what this issue is actually talking about, I'm a bit confused 😅 

Could you rephrase what exactly you believe should be updated/removed?

**Comment by henrik-jensen** [comment]
@ovflowd
Sorry for my convoluted way to ask for some examples using the streams API to read files of unknown length instead of using the fs.readFile functions which apparently is not recommended for this purpose (reading files of unknown length). 🙃 😀

**Comment by henrik-jensen** [comment]
@benhalverson and @ovflowd 
Thanks for the doc patches.
👍
